gradle.properties

```diff
@@ -5,5 +5,5 @@ group = org.quiltmc
 description = The mod loading component of Quilt
 url = https://github.com/quiltmc/quilt-loader
 
-asm_version = 9.3
+asm_version = 9.4
```

Comment:
Have you tested that mixin supports this version of ASM?

Reply:
I have tried this on a modded instance and it works fine with it.

Reply:
> Have you tested that mixin supports this version of ASM?

It seems to work fine on client-side but not on server-side so i undid that as a result.